crates/turborepo-lib/src/run/mod.rs

```diff
@@ -271,4 +271,137 @@ impl<'a> Run<'a> {
 
         Ok(())
     }
+
+    #[tokio::main]
```

Comment:
Why do we need this annotation? Is this a separate entrypoint?

Reply:
yeah because we're not running the `run` function.

Reply:
do we want to be more explicit here rather than just wrapping it in tokio::main? I am not sure where the function is called from but unless it is the 'main' entrypoint into a tokio app I think we should make this fn sync and spawn our own runtime